keps/sig-cli/2368-kubectl-default-container/kep.yaml

```diff
@@ -0,0 +1,43 @@
+title: kubectl default container
+kep-number: 2368
```

Comment:
Is this #2227 or #2368? :)

Reply:
2368 is used in the prr file

Reply:
Probably need to change the PR title and other parts that refer to the old issue then :) will take a look later, maybe closing the other issue and keeping only 2368 opened :)